packages/ia-components/sandbox/details/DetailsActionButtons.jsx

```diff
@@ -38,3 +108,5 @@ export default class DetailsActionButtons extends IAReactComponent {
         )
     }
 }
+
+export {DetailsFlagLI, DetailsFlags, DetailsActionButtons}
```

Comment:
**Linting**: add spaces within curly braces – https://github.com/airbnb/javascript#whitespace--in-braces

Reply:
I'm moving code over from dweb-archive I"m not going to make arbitrary whitespace changes just to match a style-guide that has only just been adopted.